packages/cli/lib/ui/serverlessFunctionLogs.js

```diff
@@ -0,0 +1,103 @@
+const moment = require('moment');
```

Comment:
This serverless logging functionality is duplicated in here and in the serverless-dev-runtime package. It's a little gross, but ideally we can deprecate that package soon in favor of the developer projects version of local dev for serverless functions.